src/v2/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx

```diff
@@ -22,7 +22,12 @@ export const SettingsEditProfileRouteFragmentContainer = createFragmentContainer
   {
     me: graphql`
       fragment SettingsEditProfileRoute_me on Me {
+        ...UserInformation_me
+        email
         name
+        paddleNumber
+        phone
+        internalID
       }
```

Comment:
This stuff should be in the `UserInformation` fragment, not here.

Reply:
Removed